### ISSUE TITLE
NTR: Pancreatic biopsy: pancreatic islet beta cells iron deposition (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9232]
New term request Pancreatic biopsy: pancreatic islet beta cells iron deposition (elthat_110418035726):   UMLS: 1858584

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000077
- label: Pancreatic islet beta cell iron deposition
- parents: [Abnormal pancreas morphology (HP:0012090)](https://hpo.jax.org/app/browse/term/HP:0012090)
- pmid: PMID:36882414
- def: Accumulation of abnormally high amounts of iron in beta cells of the pancreas.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9232
- synonyms:
